src/test/java/teammates/test/cases/browsertests/InstructorCourseEditPageUiTest.java

```diff
@@ -34,6 +34,19 @@ protected void prepareTestData() {
         courseId = testData.courses.get("InsCrsEdit.CS2104").getId();
     }
 
+    /**
+     * Verifies that changes to instructor are successfully saved to the database
+     * and that the success message is displayed in the browser.
+     */
+    private void verifyInstructorEditSuccess(int instructorIndex, String newName, String newEmail,
```

Comment:
1. Given the expanded scope of the method, may be the name should be `editInstructorSuccessfully()`? The header comment need to change accordingly.
2. We usually put the utility method somewhere below where it is used, not above.